instances/widgets.treasury-factory.near/widget/pages/asset-exchange/CreateRequest.jsx

```diff
@@ -210,27 +210,23 @@ return (
           onCancel: () => setShowCancelModal(true),
           onSubmit: (args) => {
             // Format the 1Click proposal with encoded metadata
+            // Include full swap details in notes for table view clarity
+            const sourceInfo = args.tokenInBlockchain
+              ? ` (${args.tokenInBlockchain})`
+              : "";
             const proposalDescription = encodeToMarkdown({
               proposal_action: "asset-exchange",
-              notes: `1Click Cross-Network Swap
-
-Swap Details:
-- Amount In: ${args.quote.amountInFormatted} ${args.tokenInSymbol}
-- Amount Out: ${args.quote.amountOutFormatted} ${args.tokenOut}
-- Destination Network: ${args.networkOut}
-- Time Estimate: ${args.quote.timeEstimate} minutes
-- Quote Deadline: ${new Date(args.quote.deadline).toLocaleString()}
-
-Deposit Address: ${args.quote.depositAddress}
-
-1Click Service Signature: ${args.quote.signature}
-
-This proposal authorizes transferring tokens to 1Click's deposit address.
-1Click will execute the cross-network swap and deliver the swapped tokens back to the treasury's NEAR Intents account.`,
+              notes: `1Click Cross-Network Swap: ${args.quote.amountInFormatted} ${args.tokenInSymbol}${sourceInfo} → ${args.quote.amountOutFormatted} ${args.tokenOut} (${args.networkOut}). This proposal authorizes transferring tokens to 1Click's deposit address for cross-network swap execution.`,
```

Comment:
is there any specific reason we are showing this in notes? because we already show all the details in the table/page using the other description keys we pass.